tgui/packages/tgui-panel/audio/NowPlayingWidget.js

```diff
@@ -30,7 +30,10 @@ export const NowPlayingWidget = (props, context) => {
               'overflow': 'hidden',
               'text-overflow': 'ellipsis',
             }}>
-            {'An admin-played soundtrack.'}
+            {
```

Comment:
Why aren't we just restoring it to the upstream `{title || 'Unknown Track'}`?

Reply:
Yea that makes the most sense, so i dont have to worry about the linter fighting with me